src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -573,6 +573,9 @@
         - name: --enable-cost-analysis
           type: bool
           short-summary: Enable exporting Kubernetes Namespace and Deployment details to the Cost Analysis views in the Azure portal. For more information see aka.ms/aks/docs/cost-analysis.
+        - name: --node-provisioning-mode
+          type: string
+          short-summary: Sets the node provisioning mode of the cluster. Valid values are "Auto" and "Manual". For more information on "Auto" mode see aka.ms/aks/nap.
```

Comment:
Please use the first-person voice in the help message
```suggestion
          short-summary: Set the node provisioning mode of the cluster. Valid values are "Auto" and "Manual". For more information on "Auto" mode see aka.ms/aks/nap.
```